iothub_client/inc/iothub_device_client_ll.h

```diff
@@ -378,6 +378,88 @@ typedef struct IOTHUB_CLIENT_CORE_LL_HANDLE_DATA_TAG* IOTHUB_DEVICE_CLIENT_LL_HA
      */
      MOCKABLE_FUNCTION(, IOTHUB_CLIENT_RESULT, IoTHubDeviceClient_LL_UploadMultipleBlocksToBlob, IOTHUB_DEVICE_CLIENT_LL_HANDLE, iotHubClientHandle, const char*, destinationFileName, IOTHUB_CLIENT_FILE_UPLOAD_GET_DATA_CALLBACK_EX, getDataCallbackEx, void*, context);
 
+     /**
+     * @brief    This API creates a context for a new blob upload to Azure Storage.
```

Comment:
Same here and for all other public API: we shouldn't start using "This API".

Reply:
I think this comment is outdated. The latest documentation is written as you suggested.